docs/connector-development/tutorials/cdk-api-source/config-based/1-create-source.md

```diff
@@ -0,0 +1,27 @@
+# Step  1: Create the Source
+
+Let's start by cloning the Airbyte repository
+
+```
+git clone git@github.com:airbytehq/airbyte.git
```

Comment:
the instructions depend on merging https://github.com/airbytehq/airbyte/pull/14552 and https://github.com/airbytehq/airbyte/pull/14923